packages/perseus/src/widgets/radio/radio-component.tsx

```diff
@@ -243,6 +256,84 @@ class Radio extends React.Component<Props> implements Widget {
         return _getPromptJSON(this.props, userInput);
     }
 
+    /**
```

Comment:
We definitely don't want to bring back `showRationalesForCurrentlySelectedChoices` or `deselectIncorrectSelectedChoices`! I'm not sure how that got mixed up in your pr work.